Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN opam pin add -yn current_docker.dev "./ocurrent" && \
     opam pin add -yn dockerfile.dev "./ocaml-dockerfile" && \
     opam pin add -yn dockerfile-opam.dev "./ocaml-dockerfile" && \
     opam pin add -yn ocluster-api.dev "./ocluster"
-COPY --chown=opam ocaml-ci.opam ocaml-ci-service.opam ocaml-ci-api.opam ocaml-ci-solver.opam /src/
+COPY --chown=opam ocaml-ci.opam ocaml-ci-web.opam ocaml-ci-service.opam ocaml-ci-api.opam ocaml-ci-solver.opam /src/
```

Comment:
This shouldn't include `ocaml-ci-web.opam`

Reply:
Ah yep sorry that survived the final cleanup (I was trying to figure out why `test` wasn't able to find `ocaml-ci-web.representation`)